content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,516 @@
+
+![GLBR architecture 2](/img/glbr-architecture-overview-2.png)
+
+*The above figure display resources that are parts of the hypervisor and on-premises components that can be protected.*
+
+> ***Note:*** 
+> In the current HPE Developer website, there is a single resource categorized as v1alpha1 that will be used to register, unregister, and update the hypervisor-manager. The current supported hypervisor-manager is VMware vCenter, and it contains the cluster, virtual machines, and datastore used for the source of data-protection. The hypervisor-manager enabled HPE GreenLake Backup and Recovery to discover the vCenter which control the assets for data protection. This a crucial discovery will lead to application of protection-policy, setup the schedule, assign the protection-store where the copy protections will be allocated, and rest of the data protection operations. 
+
+### What about the components in HPE GreenLake Backup and Recovery that are not mentioned above?
+
+There are other resources exist in the HPE GreenLake for Backup and Recovery that are not mentioned above; however, they are available in the user-interface for HPE GreenLake for Backup and Recovery. A couple of those resources are protection policy, and protection group. The protection group is used to consolidate multiple number of assets with a particular protection policy. On the other hand, the protection-policy is a resource to consolidate the schedule of protection-jobs and the flow of the recovery points at different tiers of protection-store. Together, both protection policy and protection group deliver the management of the Recovery Point Objective for data protection using HPE GreenLake for Backup and Recovery.
+
+![Protection policy for vmware schedule](/img/protection-policy-ui-for-vmware-scheduling.png)
+
+*The above figure display resources for schedule and the flow of the recovery-points part of the protection policy.*
+
+Additionally, there is also a resource called a protection job that is not shown in the user-interface; however, it is required to perform the operation of data protection. You will need to manipulate protection jobs to create a protection or to recover an existing recovery point. Additionally, the protection-job is the resource that can be manipulated to suspend or to resume a protection schedule. 
```

Comment:
Do you mean "not shown" in the UI or "shown" in the UI? It is not clear to me because the image below seems to show protection jobs.